src/InternPool.zig

```diff
@@ -5985,6 +5986,8 @@ pub const Tag = enum(u8) {
         flags: Flags,
         owner_nav: Nav.Index,
         zir_index: TrackedInst.Index,
+        location_or_descriptor_set: u32,
```

Comment:
this is definitely ugly, haven't seen other examples of large tagged unions in the intern pool so I'm not sure what the preferred pattern here is